audio_to_text_summarization_using_openai/main.bal

```diff
@@ -53,17 +47,17 @@ public function main(string podcastURL) returns error? {
     };
 
     // Summarizes the text using OpenAI text completion API
-    text:Client openAIText = check new ({auth: {token: openAIToken}});
-    text:CreateCompletionResponse completionRes = check openAIText->/completions.post(textCompletionReq);
+    final chat:Client openAIChat = check new({auth: {token:openAIToken}});
+    chat:CreateChatCompletionResponse completionRes = check openAIChat->/chat/completions.post(request);
 
-    string? summerizedText = completionRes.choices[0].text;    
+    string? summerizedText = completionRes.choices[0].message.content;    
     if summerizedText is () {
 	    return error("Failed to summarize the given audio.");
     }
     io:println("Summarized text: ", summerizedText);
  
     // Tweet it out!
-    twitter:Client twitter = check new (twitterConfig);
-    var tweet = check twitter->tweet(summerizedText);
+    final twitter:Client twitter = check new({auth: {token:token}});
+    twitter:TweetCreateResponse tweet = check twitter->/tweets.post(payload = {text: summerizedText});
```

Comment:
do we need named args in here? Can't we do
```suggestion
    twitter:TweetCreateResponse tweet = check twitter->/tweets.post({text: summerizedText});
```
?

Reply:
> do we need named args in here? Can't we do
> 
> ?

We can. I'll make the necessary change